php/libraries/Database.class.inc

```diff
@@ -28,6 +31,16 @@ define("DEBUG", false);
  */
 class Database
 {
+    /**
+     * Constants determining whether HTML characters will be escaped before
+     * being inserted into the Database.
+     *
+     * @access private
```

Comment:
These are not private. They're used as arguments to public methods.

Reply:
You're right, this is left over from a previous commit.